bin/dev

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
```

Comment:
- [ ] While running bin/dev I'm getting the following error. Kindly fix it

![image](https://github.com/Booze33/hello-rails--react/assets/93630700/879104ce-3659-4ced-8963-186427d6d28c)